### PR TITLE
Add pg esm test

### PR DESCRIPTION
**Pull request by SevInf**
Broken until prisma/prisma#21983 is merged.

### CODE REVIEW

File: driver-adapters/postgresql-node-basic-esm/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "postgresql-node-basic",
```
Comment (Jolg42) [inline driver-adapters/postgresql-node-basic-esm/package.json]:
```suggestion
  "name": "postgresql-node-basic-esm",
```

File: driver-adapters/postgresql-node-basic-esm/prepare.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -eu
+
+export PRISMA_TELEMETRY_INFORMATION='ecosystem-tests driver-adapters postgresql-node-basic'
```
Comment (Jolg42) [inline driver-adapters/postgresql-node-basic-esm/prepare.sh]:
```suggestion
export PRISMA_TELEMETRY_INFORMATION='ecosystem-tests driver-adapters postgresql-node-basic-esm'
```

**Review by Jolg42 — APPROVED** [review]

**Comment by Jolg42** [comment]
@SevInf the yml needs an update
```
There are some folders in ecosystem-tests that are not used in test.yaml or optional-test.yaml workflows
[ 'driver-adapters/postgresql-node-basic-esm' ]
```

File: driver-adapters/postgresql-node-basic-esm/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  },
+  "devDependencies": {
+    "@jest/globals": "29.7.0",
+    "@types/pg": "^8.10.2",
+    "jest": "28.1.3",
+    "prisma": "5.7.0-integration-fix-pg-esm.1"
+  },
+  "dependencies": {
+    "@prisma/adapter-pg": "5.7.0-integration-fix-pg-esm.1",
+    "@prisma/client": "5.7.0-integration-fix-pg-esm.1",
```
Comment (SevInf) [inline driver-adapters/postgresql-node-basic-esm/package.json]: Revert to dev before merging

**Pull request merged**